graalvm/suite.yml

```diff
@@ -6,7 +6,7 @@ connections:
   - name: "Blazegraph"
     version: "1.1.1"
     dataset: "DatasetName"
-    endpoint: "http://localhost:9999/blazegraph/sparql"
+    endpoint: "https://dbpedia.org/sparql"
```

Comment:
We've just had an issue with the wikidata endpoint because it requires the [User-Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) to be set, see: https://foundation.wikimedia.org/wiki/Policy:User-Agent_policy

Reply:
The Apache HttpClient automatically sends that header.